### ISSUE TITLE
Todo example does not work when coroutines are added to the main module

### ISSUE THREAD

**Issue by arkivanov** [JetBrains/compose-jb#103]
STR: 

- add coroutines to the `main` module: `implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.1")`
- run desktop app

Actual: the window is blank
Expected: a working app

I see that all Composable functions are triggered with valid data but nothing is rendered. Looks like the `Crossfade` function becomes broken. Removing `Crossfade` fixes the issue:
```
Index: examples/todoapp/common/root/src/commonMain/kotlin/example/todo/common/root/integration/TodoRootImpl.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- examples/todoapp/common/root/src/commonMain/kotlin/example/todo/common/root/integration/TodoRootImpl.kt	(revision 1b294335a9394ba32c042e5aaee5021a7a288c91)
+++ examples/todoapp/common/root/src/commonMain/kotlin/example/todo/common/root/integration/TodoRootImpl.kt	(date 1605354747181)
@@ -63,9 +63,7 @@
     @Composable
     override fun invoke() {
         router.state.children { child, configuration ->
-            Crossfade(currentChild = child, currentKey = configuration) { currentChild ->
-                currentChild()
-            }
+            child()
         }
     }
 
```

**Comment by CarsonRedeye** [comment]
I noticed on my app adding coroutines makes TextFields very buggy and very few keystrokes are recognised

**Comment by CarsonRedeye** [comment]
FYI `org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9` works OK with TextFields, but org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.0-M1` or higher does not.

**Comment by igordmn** [comment]
Fixed in 0.2.0-build128. I checked that `todoapp` works if we add coroutines 1.4.1 in `main` module.

Was fixed in:
https://android-review.googlesource.com/c/platform/frameworks/support/+/1495139

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6513">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
